src/anm/player.js

```diff
@@ -437,7 +438,18 @@ Player.prototype.load = function(arg1, arg2, arg3, arg4) {
     }
 
     return player;
-}
+};
+
+/**
+ * Connect audio/video elements with player events
+ */
+var initMedia = function(player) {
+    player.anim.traverse(function(el) {
+        if (el.type === C.ET_AUDIO || el.type === C.ET_VIDEO) {
+            (el.$audio || el.$video).connect(player);
+        }
+    });
```

Comment:
And it should also unsubscribe on `S_LOAD` event (new animation loaded into Player), by the way. If `S_START`/`S_END` events will fire from `Animation` class instead of `Player`, this won't be required, since `player.anim` is replaced on load.